### PR TITLE
Update activity vignette

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: R/ab_scenario.R [hunk 1]
```diff
@@ -283,7 +283,8 @@ ab_sf = function(
   sf_data = subset(trip_data, select = -c(origin, destination))
   sf_linestring = sf::st_sf(
     sf_data,
-    geometry = linestrings
+    geometry = linestrings,
+    crs = 4326
```
Comment (natesheehan) [inline R/ab_scenario.R:287]: Sure 👍

File: vignettes/activity.Rmd [hunk 2]
```diff
@@ -82,16 +82,24 @@ od = tibble::tribble(
...
+```{r}
 od_sf = od::od_to_sf(od, places_sf)
 plot(od_sf["departure"], reset = FALSE, key.pos = 1, lwd = 6:2)
 plot(places_sf$geometry, pch = places$name, add = TRUE, cex =2)
 # mapview::mapview(od_sf["departure"])
 ```
 
-```{r, eval=FALSE}
-od::od_coordinates(od_sf)
+As an aside, another way of representing the spatial attributes of the OD data with with four columns with 'X' and 'Y' coordinates for both origins and destinations:
```
Comment (natesheehan) [inline vignettes/activity.Rmd:96]: "with with"

File: vignettes/activity.Rmd [hunk 3]
```diff
@@ -146,9 +168,14 @@ That results in the following file:
 
 ```
 
-What do the first 2 trips look like?
+You can check the 'round trip' conversion of this JSON representation back into the data frame representation as follows:
+
```
Comment (natesheehan) [inline vignettes/activity.Rmd:172]: Nice!

**Review by natesheehan — COMMENTED** [review]
Great stuff Robin, especially after your 2 am wake up! One copy thing I noticed but everything is reproducible on my side. Happy for you to merge before or after the copy change!

**Comment by Robinlovelace** [comment]
> Great stuff Robin, especially after your 2 am wake up! One copy thing I noticed but everything is reproducible on my side. Happy for you to merge before or after the copy change!

That's what siestas are for!

File: R/ab_scenario.R [hunk 4]
```diff
@@ -189,6 +189,7 @@ ab_json = function(
       "if the original input was in 10,000th of a second (used internally by A/B Street)"
       )
   }
+  desire_lines$departure = desire_lines$departure * 10000
```
Comment (natesheehan) [inline R/ab_scenario.R:192]: 👍

**Review by natesheehan — APPROVED** [review]

**Pull request merged**